### PR TITLE
November22 01

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+      - [Onboarding Vendors](onboard-vendors-to-techpass)
 - [Log in to TechPass Portal](log-into-techpass-portal)
-  - [Edit Profile](edit-profile-using-non-se-gsib)
-- [Access SGTS services using TechPass](access-sgts-services-using-techpass)  
-- **Vendor**
-  - [Onboarding Vendors](onboard)
-  - [Support](/support/overview.md)
+- [Edit Profile](edit-profile-using-non-se-gsib)
+- [Access SGTS services using TechPass](access-sgts-services-using-techpass)
+- [Re set TechPass MFA](re-set-techpass-mfa-for-new-device)
```
Comment (yilunyou) [inline docs/_sidebar.md:13]: Reset

File: docs/access-sgts-services-using-techpass.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 TechPass users can use their TechPass to securely and seamlessly access SGTS services such as SEED, GCC2.0, Cloud File Transfer (CFT), SHIP-HATS and Developer portal using their GMD and GSIB devices.
 
-This article guides you on how to use TechPass to access SGTS resources. For documentation purpose, we show you how to access Developer portal documents that can only be accessed using TechPass.
+This article guides you on how to use TechPass to access SGTS resources.
 
 - [Log in with TechPass for vendors](#to-log-in-with-techpass-for-vendor)
-- [Log in with TechPass for public officers](#to-log-in-with-techpass-for-public-officers)
+- [Log in with TechPass for public officers](#to-log-in-with-techpass-for-public-officer)
```
Comment (yilunyou) [inline docs/access-sgts-services-using-techpass.md:8]: [Log in with TechPass for public officer]

File: docs/onboard-vendors-to-techpass.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+# Onboarding Vendors
+This section guides how vendors or contractors working on Gov ICT Projects can get a TechPass account and onboard in to it.
+
+## How to get a TechPass account for vendors or contractors?
+
+If you are a vendor or contractor who does not have an active TechPass account, request one from your project manager or reporting officer. Make sure you provide your official email address, handphone number and your project name in your request. Your project manager or reporting officer in turn will work with the engaging government agency to set up a TechPass account for you.
+
+?>
+  You can use this TechPass account for projects hosted by all government agencies.<br><br>
```
Comment (yilunyou) [inline docs/onboard-vendors-to-techpass.md:9]: You can use TechPass account to access SGTS products and services

File: docs/onboard-vendors-to-techpass.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+
+## Onboarding vendors
+When a TechPass account is created, you will be notified about it in an email. This notification will have a unique TechPass username for you and an initial password will be texted to your handphone.
+
+Complete the following steps:
+
+<details> <summary> Sign in using TechPass username and initial password</summary>
+
+  1. Go to the web address(url) provided by your project manager or reporting officer to sign in to TechPass.
```
Comment (yilunyou) [inline docs/onboard-vendors-to-techpass.md:20]: 1. Go to the web address(url) provided by your project manager or reporting officer to sign in to SGTS service with your TechPass account.

File: docs/onboard-vendors-to-techpass.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  13. Click **Next**.
+
+      <kbd>![vendor-mfa-8](assets/support/vendor-mfa-8.png)</kbd>
+  14. When you see a success message, click **Done**.
+      <kbd>![vendor-mfa-9](assets/support/vendor-mfa-9.png)</kbd>
+
+      Now you will be prompted to re set your initial password.
+</details>
+
+<details> <summary> Re set your initial password of TechPass</summary>
```
Comment (yilunyou) [inline docs/onboard-vendors-to-techpass.md:71]: Reset

File: docs/onboard-vendors-to-techpass.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+      <kbd>![vendor-mfa-7](assets/support/vendor-mfa-7.png)</kbd>
+
+  13. Click **Next**.
+
+      <kbd>![vendor-mfa-8](assets/support/vendor-mfa-8.png)</kbd>
+  14. When you see a success message, click **Done**.
+      <kbd>![vendor-mfa-9](assets/support/vendor-mfa-9.png)</kbd>
+
+      Now you will be prompted to re set your initial password.
```
Comment (yilunyou) [inline docs/onboard-vendors-to-techpass.md:68]: reset

File: docs/onboard-vendors-to-techpass.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+</details>
+
+<details> <summary> Re set your initial password of TechPass</summary>
+
+  15. Enter your **initial password**, **new password** and re type new password to confirm.
```
Comment (yilunyou) [inline docs/onboard-vendors-to-techpass.md:73]: retype or re-type?

File: docs/re-set-techpass-mfa-for-new-device.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+# Re set TechPass MFA (New mobile device)
```
Comment (yilunyou) [inline docs/re-set-techpass-mfa-for-new-device.md:1]: reset

File: docs/support/account.md [hunk 9]
```diff
@@ -1,115 +1,32 @@
...
+4. I have lost the mobile device which I used for MFA authentications. What should I do?</summary>
 
-1. I am a vendor and my TechPass account has got locked. How to unlock my TechPass account?
+**If you are a vendor**:
 
-  Vendor's TechPass account will be locked after five consecutive unsuccessful login attempts. To unlock, go to https://passwordreset.microsoftonline.com/ and follow the on-screen instructions.
+i. Raise a [service request](https://go.gov.sg/techpass-sr) to contact our technical support to remove the MFA configured for your TechPass account.
 
-2. What is your password reset policy for TechPass accounts?
+ii. Once this done you will be notified. Proceed to [re-set-techpass-mfa-for-new-device](re-set-techpass-mfa-for-new-device) using your new mobile device.
```
Comment (yilunyou) [inline docs/support/account.md:20]: if there's a filename change, change this too: re-set-techpass-mfa-for-new-device

File: docs/support/account.md [hunk 10]
```diff
@@ -1,115 +1,32 @@
...
 
-  For vendors, we follow the [password policy of Azure Active Directory](https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-policy#administrator-password-policy-differences).  Hence vendors will receive a password expiry notification accordingly. You can [reset password](https://passwordreset.microsoftonline.com/) by following the on-screen instructions.
+ **If you are a public officer**:
 
-  However, if you are a public officer, you will be notified to reset your GSIB password as per the WOG password policies. If there are any issues with this, please contact your organisation's Agency Facility Management (AFM).
+i. Contact your Agency Facility Management (AFM) to  remove the MFA configured for your WOG account and raise a [service request](https://go.gov.sg/techpass-sr) to remove the MFA configured for your TechPass account.
 
+ii. Once these are completed, re set MFA for [WOG account](re-set-security-verification-for-wog-account) and [TechPass account](re-set-techpass-mfa-for-new-device) using your new mobile device.
```
Comment (yilunyou) [inline docs/support/account.md:26]: reset

Comment (SenthilKumarSivakami) [reply docs/support/account.md:26]: done

File: docs/support/account.md (hunk 9 above)
Comment (SenthilKumarSivakami) [reply docs/support/account.md:20]: done .

File: docs/re-set-techpass-mfa-for-new-device.md (hunk 8 above)
Comment (SenthilKumarSivakami) [reply docs/re-set-techpass-mfa-for-new-device.md:1]: Done

File: docs/onboard-vendors-to-techpass.md (hunk 7 above)
Comment (SenthilKumarSivakami) [reply docs/onboard-vendors-to-techpass.md:73]: changed it as "retype".

File: docs/onboard-vendors-to-techpass.md (hunk 6 above)
Comment (SenthilKumarSivakami) [reply docs/onboard-vendors-to-techpass.md:68]: Done

File: docs/onboard-vendors-to-techpass.md (hunk 5 above)
Comment (SenthilKumarSivakami) [reply docs/onboard-vendors-to-techpass.md:71]: Done

File: docs/onboard-vendors-to-techpass.md (hunk 4 above)
Comment (SenthilKumarSivakami) [reply docs/onboard-vendors-to-techpass.md:20]: Done.  also updated line 9.

File: docs/onboard-vendors-to-techpass.md (hunk 3 above)
Comment (SenthilKumarSivakami) [reply docs/onboard-vendors-to-techpass.md:9]: Done

File: docs/access-sgts-services-using-techpass.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/access-sgts-services-using-techpass.md:8]: Updated, the bookmarks which I said were these 2 lines, I have fixed them now.

File: docs/_sidebar.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/_sidebar.md:13]: Done.

**Pull request merged**